server/src/main/java/org/apache/iotdb/db/sync/conf/SyncPathUtil.java

```diff
@@ -39,13 +39,13 @@ public static String getSenderPipeDir(String pipeName, long createTime) {
         + String.format("-%s-%d", pipeName, createTime);
   }
 
-  public static String getSenderHistoryPipeDataDir(String pipeName, long createTime) {
+  public static String getSenderHistoryPipeLogDir(String pipeName, long createTime) {
     return getSenderPipeDir(pipeName, createTime)
```

Comment:
make directory structure on the sender side same as on the receiver side

Reply:
done